orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTaskSpec.groovy

```diff
@@ -250,6 +258,10 @@ class FindImageFromClusterTaskSpec extends Specification {
         ]
       ]
     ]
+
+    cloudProvider || findCalls
```

Comment:
curious why do you need `||` vs `|`?

Reply:
They're interchangeable. There's a convention to use || to separate an output or result parameter vs. input but not sure if it really makes sense here.